cowprotocol/cowswap
# 1020/replace blockscout with gnosisscan recipient explorer link

# Summary

Follow up to #1021 addressing [comment #1](https://github.com/cowprotocol/cowswap/pull/1021#issuecomment-1237156406)

> Recipient link navigates to Etherscan in the GC (not reported anywhere, I just have found it, but it is on Prod now)

  # To Test

1. On gnosis chain, enable expert mode or add recipient toggle on settings
2. Add a recipient and insert a valid address
* The link on top right of the modal should take you to gnosisscan rather than etherscan
![Screen Shot 2022-09-05 at 17 30 01](https://user-images.githubusercontent.com/43217/188489639-b358d0b9-e07d-4def-9ddd-54988126a9e7.png)
